WalletWasabi/Helpers/ImportWalletHelper.cs

```diff
@@ -52,8 +52,8 @@ private static KeyManager GetKeyManagerByWasabiJson(WalletManager manager, strin
 
 	private static KeyManager GetKeyManagerByColdcardJson(WalletManager manager, JObject jsonWallet, string walletFullPath)
 	{
-		var xpubString = jsonWallet["ExtPubKey"].ToString();
-		var mfpString = jsonWallet["MasterFingerprint"].ToString();
+		var xpubString = jsonWallet["ExtPubKey"]!.ToString();
+		var mfpString = jsonWallet["MasterFingerprint"]!.ToString();
```

Comment:
Here you don't know if those fields are null. What you are doing here is not fixing a problem but creating one. The only why to be sure they are not null is by checking their values.